hikvision_tftpd.py

```diff
@@ -73,6 +73,17 @@ def _bind(self, addr):
             raise
         return sock
 
+    def set_block_size(self, block_size):
+        print 'Setting block size to %d' % block_size
+        self.BLOCK_SIZE = block_size
```

Comment:
could you make BLOCK_SIZE lowercase, as it's no longer a constant? maybe also initialize it in the constructor rather than at the class level for clarity.